File: Spotter/UI/spotter_desktop/lib/core/models/ticket_type_insert_request.dart
class TicketTypeInsertRequest {
  final int eventId;
  final String name;
  final double price;
  final int totalQuantity;
  final int typeEnum;

  TicketTypeInsertRequest({
    required this.eventId,
    required this.name,
    required this.price,
    required this.totalQuantity,
    required this.typeEnum,
  });

  Map<String, dynamic> toJson() => {
        'eventId': eventId,
        'name': name,
        'price': price,
        'totalQuantity': totalQuantity,
        'typeEnum': typeEnum,
      };
}

File: Spotter.Model/Requests/TicketTypeInsertRequest.cs
using Spotter.Model.Enums;

namespace Spotter.Model.Requests
{
    public class TicketTypeInsertRequest
    {
        public int EventId { get; set; }
        public string Name { get; set; } = string.Empty;
        public decimal Price { get; set; }
        public int TotalQuantity { get; set; }
        public TicketTypeEnum TypeEnum { get; set; }
    }
}
